bin/aursearch

```diff
@@ -30,16 +30,13 @@ split() {
 }
 
 getjson() {
```

Comment:
I added the -P flag and set the default to use by=name search, that's why the two differ.